xline/src/server/xline_server.rs

```diff
@@ -86,62 +137,6 @@ impl XlineServer {
     }
 }
 
-/// Command to run consensus protocal
-#[derive(Debug, Serialize, Deserialize)]
-pub(crate) struct Command {
-    /// Key of request
-    key: String,
-    ///
-    /// Encoded request data
-    request: Vec<u8>,
-}
-
-impl Command {
-    /// New `Command`
-    pub(crate) fn new(key: String, request: Vec<u8>) -> Self {
-        Self { key, request }
-    }
-
-    /*
-    /// Get key of `Command`
-    pub(crate) fn key(&self) -> &String {
-        &self.key
-    }
-    /// Get request of `Command`
-    pub(crate) fn request(&self) -> &Vec<u8> {
-        &self.request
-    }
-    */
-
-    /// Consume `Command` and get ownership of each field
-    pub(crate) fn unpack(self) -> (String, Vec<u8>) {
-        let Self { key, request } = self;
-        (key, request)
-    }
-}
-
-/// Execution Request
-#[derive(Debug)]
-pub(crate) struct ExecutionRequset {
-    /// Command to execute
-    command: Command,
-    /// Execution result sender
-    notifier: Sender<ResponseOp>,
-}
-
-impl ExecutionRequset {
-    /// New `ExecutionRequest`
-    pub(crate) fn new(command: Command) -> (Self, Receiver<ResponseOp>) {
-        let (notifier, receiver) = channel();
-        (Self { command, notifier }, receiver)
-    }
-    /// Consume `ExecutionRequest` and get ownership of each field
-    pub(crate) fn unpack(self) -> (Command, Sender<ResponseOp>) {
-        let Self { command, notifier } = self;
-        (command, notifier)
-    }
-}
-
 #[tonic::async_trait]
 impl Kv for XlineRpcServer {
```

Comment:
Can we avoid repeated code via generic?

Reply:
I will move these common codes to a function.

Reply:
> Will you replace most panics with proper Error in the future?

Yes. we should only panic on unrecoverable error.